ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -18,15 +18,15 @@ Item {
     property bool hideSignPhraseModal: false
 
     function showSigningPhrasePopup(){
-        if(!hideSignPhraseModal && !localAccountSensitiveSettings.hideSignPhraseModal){
+        if(!hideSignPhraseModal && !RootStore.hideSignPhraseModal){
             signPhrasePopup.open();
         }
     }
 
     SignPhraseModal {
         id: signPhrasePopup
         onRemindLaterClicked: hideSignPhraseModal = true
-        onAcceptClicked: localAccountSensitiveSettings.hideSignPhraseModal = true
+        onAcceptClicked: RootStore.hideSignPhraseModal = true
```

Comment:
This `hideSignPhraseModal` mapped/aliased to `localAccountSensitiveSettings`?

We use setter functions to store this value in a config file (in `Status/qt/[account]`), so we need to make sure this still works when this value is changed.

Reply:
Also, I know the way Sale created the settings was meant in a "globally available" way, so maybe passing through the Store for them is "overkill"?

Reply:
@PascalPrecht if you check on wallet's RootStore there is code for this:
```
onHideSignPhraseModalChanged: {
    accountSensitiveSettings.hideSignPhraseModal = hideSignPhraseModal;
}
```
I could introduce another function for this if you want.

@jrainville yes correct, however this is way is not flexible if we want to have the UI running with a simulation backend, we cannot be introducing dependencies, that's why we use the stores to encapsulate things there and have a single place to update.

Reply:
Thanks for clarifying! @alexandraB99 if this keeps the same logic around, this is fine by me as is!

Reply:
@PascalPrecht I switched to use a setter function :)